src/explorer/SiteTreeItem.ts

```diff
@@ -101,9 +102,7 @@ export abstract class SiteTreeItem implements IAzureParentTreeItem {
         this._logStream = kuduClient.getLogStream().on('data', chunk => {
             this._logStreamOutputChannel.append(chunk.toString());
         }).on('error', err => {
-            util.sendTelemetry('ConnectToLogStreamError', { name: err.name, message: err.message });
-            this._logStreamOutputChannel.appendLine('Error connecting to log-streaming service:');
-            this._logStreamOutputChannel.appendLine(err.message);
```

Comment:
Are these covered by `throw(err)` you just added?

Reply:
Not exactly.  We were trying to streamline the error handling, but see the thread above to see why that won't work :(